libmscore/edit.cpp

```diff
@@ -2252,12 +2252,15 @@ void Score::deleteMeasures(MeasureBase* is, MeasureBase* ie, bool preserveTies)
                         if (!s)
                               s = mAfterSel->undoGetSegment(SegmentType::TimeSig, mAfterSel->tick());
 
-                        TimeSig* nts = new TimeSig(this);
-                        nts->setTrack(staffIdx * VOICES);
-                        nts->setParent(s);
-                        nts->setFrom(lastDeletedForThisStaff);
-                        nts->setStretch(nts->sig() / mAfterSel->timesig());
-                        score->undoAddElement(nts);
+                        auto track = staffIdx * VOICES;
+                        if (track < static_cast<int>(s->elist().size())) {
```

Comment:
maybe better change that `auto` to `size_t`

Reply:
probably right. no casting needed then